R/getFullDesign.R

```diff
@@ -32,9 +32,6 @@
 #' @export
 getFullDesign = function(formula, data, center = TRUE){
 
-	# Fill in the data argument with stats::model.frame
```

Comment:
Is it not possible to include this here? Since getFullDesign() is a standalone function, it would be great if 'data' were an optional argument.